kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/steps/KafkaSteps.java

```diff
@@ -61,15 +68,21 @@ public static void createTopic(String deployNamespace, String topicName, String
      * @param replicas the replicas
      * @param compressionType the compression type
      */
-    public static void createTopic(String deployNamespace, String topicName, String bootstrap, int partitions, int replicas, @NonNull CompressionType compressionType) {
+    public static void createTopic(String deployNamespace, String topicName, String bootstrap, int partitions, int replicas,
+                                   @NonNull CompressionType compressionType) {
         LOGGER.atDebug().setMessage("Creating '{}' topic").addArgument(topicName).log();
         String name = Constants.KAFKA_ADMIN_CLIENT_LABEL + "-create";
         List<String> args = new ArrayList<>(
-                List.of(TOPIC_COMMAND, "create", BOOTSTRAP_ARG + bootstrap, "--topic=" + topicName, "--topic-partitions=" + partitions,
+                List.of(TOPIC_COMMAND, "create", BOOTSTRAP_ARG + bootstrap, TOPIC_ARG + topicName, "--topic-partitions=" + partitions,
                         "--topic-rep-factor=" + replicas));
```

Comment:
For consistency we should also pull out `--topic-partitions=` and `"--topic-rep-factor="` as constants.